Xamarin.Forms.Platform.UAP/ScrollViewRenderer.cs

```diff
@@ -227,9 +232,17 @@ void UpdateVerticalScrollBarVisibiilty()
 
 		void UpdateHorizontalScrollBarVisibility()
 		{
+			var horizontalVisibility = Element.HorizontalScrollBarVisibility;
+
+			if (horizontalVisibility == ScrollBarVisibility.Default)
+			{
+				UpdateOrientation();
+				return;
+			}
```

Comment:
I'll take a look at it again...it's been a while since I did this so I'm a little rusty on it

Reply:
@paymicro After walking through the code again, UpdateOrientation is called here to reset the scroll bar visibility to the default values for the current orientation.  I'll add a comment as the method name it's probably isn't the clearest, but it makes sense to call it UpdateOrientation since it's primary purpose is to handled the Orientation property change.

Reply:
Can you add that comment and rebase? thanks

Reply:
Sure, might take me until next week to get to this as I'm on the road at a company event through next week.